spec.bs

```diff
@@ -253,8 +253,9 @@ To <dfn>check interest group permissions</dfn> given an [=origin=]
 1. If |ownerOrigin| is [=same origin=] to |frameOrigin|, then return true.
 1. Let |permissionsUrl| be the result of [=building an interest group permissions url=]
   with |ownerOrigin| and |frameOrigin|.
-1. Let |request| be the result of [=creating a request=] with |permissionsUrl|, [=create a request/accept=]
-  set to "`application/json`"|. TODO: [maybe](https://source.chromium.org/chromium/chromium/src/+/main:content/browser/interest_group/interest_group_permissions_checker.cc;drc=adac219925ef5d9c9a954d189c2e4b8852a4bbed;l=123) require cors mode here?
+1. Let |request| be the result of [=creating a request=] with |permissionsUrl|,
+  "`application/json`", and null. TODO: [maybe](https://source.chromium.org/chromium/chromium/src/+/main:content/browser/interest_group/interest_group_permissions_checker.cc;drc=adac219925ef5d9c9a954d189c2e4b8852a4bbed;l=123) require
+  cors mode here?

```

Comment:
I see you didn't add this here, so this shouldn't block merging this PR, but I wanted to point out that if you don't use CORS mode, you'll need to include an explanation of both why CORS isn't needed, and why it would be harmful to follow the web's default of enabling CORS. Otherwise you'll worry security reviewers across every browser that looks at this.

Reply:
We do use CORS as the code link indicated, so I removed this scary language otherwise it'll scare folks unnecessarily.